packages/react-composites/tests/browser/call/app/mocks/MockCallAdapter.ts

```diff
@@ -0,0 +1,237 @@
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-explicit-any
+  dispose(): void {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  joinCall(microphoneOn?: boolean): Call | undefined {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async leaveCall(forEveryone?: boolean): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async startCamera(options?: VideoStreamOptions): Promise<void> {
+    return;
+  }
+  async stopCamera(): Promise<void> {
+    return;
+  }
+  async mute(): Promise<void> {
+    return;
+  }
+  async unmute(): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  startCall(participants: string[]): Call | undefined {
+    return;
+  }
+  async startScreenShare(): Promise<void> {
+    return;
+  }
+  async stopScreenShare(): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async removeParticipant(userId: string): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async createStreamView(remoteUserId?: string, options?: VideoStreamOptions): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async disposeStreamView(remoteUserId?: string, options?: VideoStreamOptions): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async askDevicePermission(constrain: PermissionConstraints): Promise<void> {
+    return;
+  }
+
+  public async queryCameras(): Promise<VideoDeviceInfo[]> {
+    return [];
+  }
+
+  public async queryMicrophones(): Promise<AudioDeviceInfo[]> {
+    return [];
+  }
+
+  public async querySpeakers(): Promise<AudioDeviceInfo[]> {
+    return [];
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async setCamera(sourceInfo: VideoDeviceInfo, options?: VideoStreamOptions): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async setMicrophone(sourceInfo: AudioDeviceInfo): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async setSpeaker(sourceInfo: AudioDeviceInfo): Promise<void> {
+    return;
+  }
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-explicit-any
+  on(event: string): void {
+    return;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-explicit-any
+  off(event: string): void {
+    return;
+  }
+}
+
+/**
+ * Helper function to create remoteParticipants property of CallAdapterState from an array of TestRemoteParticipant
+ * @param mockRemoteParticipants - array of TestRemoteParticipant
+ * @returns Record of RemoteParticipantState
+ */
+
+const createMockRemoteParticipants = (
```

Comment:
rename this function -- it is called `createMockRemoteParticipants` but takes in an already created array called `mockRemoteParticipants`.
Judging by the logic what a suggestion for this would be:
`createMockRemoteParticipantsWithVideo`?

Reply:
Hmm actually this does more than just add video information, it also returns an object of keyed participants...

Reply:
Consider splitting into two separate functions for better clarity and to ensure that a function only has one purpose.
Instead of:
```ts
const remoteParticipants = createMockRemoteParticipants(testState.remoteParticipants);
```
This could instead be something like:
```ts
const remoteParticipants = convertTestParticipantsToCallAdapterStateParticipants(testState.remoteParticipants);
Object.values(remoteParticipants).addMockVideo(participant);
```